_i18n/en/develop.md

```diff
@@ -0,0 +1,34 @@
+As a community project, AntennaPod relies on voluntary code contributions from a wide range of people. Not just for the app itself, but also the website and related projects.
+
+# App
+You're right at it: the core of the project! Used by thousands of people across the world, your contributions are an important contribution to an open podcasting ecosystem. So, are you experienced with Java, or just getting started? We welcome your contributions in our [GitHub repository](https://github.com/AntennaPod/AntennaPod).
+
+## Where we need help
```

Comment:
Hmm, to be honest, I think open issues rather belong on GitHub, not on the website.

Reply:
The thing is: there's nowhere a clear indication of what we want help with (with priority). This short list gives a clear direction. GitHub is a massive wall of issues, thus maybe daunting to navigate and certainly not gently directing potential contributors. Also it is no guaranteed to be 'clean' because it is not curated (i.e. some feature request we might actually not want to be implemented). (And these are not issues themselves, but rather priority areas with references to the issues.)

Reply:
If you want to curate issues, we could enable the project board feature on GitHub. I had a discussion on the forum with tonytamsf and then decided that the feature creates more work than it saves for me, personally. Used differently, it could be helpful, though :)